DirectEmployers/MyJobs
# [PD-3610] Try loading report results lazily.

## What's this PR do?

It should speed up loading reporting by allowing the reports list to download without loading and parsing all the saved reports results.

## Where should the reviewer start?

There's just the lazification code.

## How should this be manually tested?

Look at PRM reports and make sure you can still use it. In particular you should be able to run and export reports.

## Any background context you want to provide?

na
